Datastore/src/Operation.php

```diff
@@ -549,10 +557,14 @@ public function mutation(
         $type,
         $baseVersion = null
     ) {
-        // If the given element is an Entity, it will use that baseVersion.
-        if ($input instanceof Entity) {
+        // If the given element is an EntityInterface, it will use that baseVersion.
+        if ($input instanceof EntityInterface) {
             $baseVersion = $input->baseVersion();
             $data = $this->entityMapper->objectToRequest($input);
+
+            if (!$input->key()) {
+                throw new \InvalidArgumentException('Base entities must provide a datastore key.');
```

Comment:
Small nit, the exception is already imported so no need for the leading slash

Reply:
I removed the import and kept/added the slash as this is a more common convention in the library!